pkg/gitconfig/gitconfig.go

```diff
@@ -0,0 +1,15 @@
+package gitconfig
+
+var (
+	// SystemConfig is the location of the (optional) system-wide config defaults file.
+	systemConfig = "/etc/gitconfig" // /etc/gopass/config
+	// GlobalConfig is the location of the (optional) global (i.e. user-wide) config file.
+	globalConfig = ".gitconfig"
+	// LocalConfig is the name of the local (per-workdir) configuration.
+	localConfig = "config"
+	// WorktreeConfig is the name of the local worktree configuration. Can be used to override
+	// a committed local config.
+	worktreeConfig = "config.worktree"
```

Comment:
I think that worktree configs are even worse, since their config is stored in the _original repo_'s `.git` folder under `.git/worktrees/nameofwt/config.worktree`

e.g. if I have a worktree named "myown" located at `/home/anomalroil/code/myown` for gopass it would look like:
```
file:/home/anomalroil/.gitconfig        commit.gpgsign=true
file:/home/anomalroil/code/gopass/.git/config    user.email=anomalroil@users.noreply.github.com
file:/home/anomalroil/code/gopass/.git/worktrees/myown/config.worktree   user.name=TEST
``` 
because in a worktree, the folder `.git` is replaced with a file `.git` specifying the location of its original gitdir:
```
$ cat .git
gitdir: /home/anomalroil/code/drand/.git/worktrees/myown
```
Is this currently taken into account?

Reply:
No, it isn't. And I wasn't aware of that. Thank you. I don't necessarily plan to support this as part of this PR, but it's definitely something that needs to be documented and should have a TODO. Will add that and then resolve this comment.

Reply:
Reading https://git-scm.com/docs/git-config#SCOPES I'm not sure if that's correct - but I haven't used this feature before so I might be wrong.